config/desktop.json

```diff
@@ -38,6 +38,7 @@
 		"gutenberg/opt-in": true,
 		"help": true,
 		"help/courses": true,
+		"jetpack/checklist/performance": true,
```

Comment:
Should also enable in `desktop-development`?

Reply:
Nice, totally 👍